src/plugins/data_view_editor/public/open_editor.tsx

```diff
@@ -85,6 +85,10 @@ export const getEditorOpener =
         {
           hideCloseButton: true,
           size: 'l',
+          maskProps: {
+            // EUI TODO: This z-index override of EuiOverlayMask is a workaround, and ideally should be resolved with a cleaner UI/UX flow long-term
+            style: 'z-index: 1003', // we need this flyout to be above the timeline flyout (which has a z-index of 1002)
```

Comment:
it would be nice to managed the z-index wars via variables but thats not necessary for this PR.

Reply:
👍 for using variables here (not blocking this PR of course), here's an example how it's done in other areas: 
https://github.com/elastic/kibana/blob/f8a6324444badb7b8eaa085283f3c7d2659e5f61/x-pack/plugins/cases/public/components/create/flyout/create_case_flyout.tsx#L47

Reply:
I would agree with @kertal . Infact, we can also do it only in `Security Solution` so that it does not interface with other areas.

For example, increase this z-index here might display it above existing flyouts? Just a food for thought, if you could double check.

Reply:
noted

Reply:
i will fix it in a separate pr, it is done inline in other places as well

Reply:
thought that it will be a 1 minutes fix but there's some prop forwarding that has to happen to get it right.

Reply:
Currently, for flyouts in timeline, it is done like below with css. You can explore this.

https://github.com/elastic/kibana/blob/85f12800bb80d959b3d1fb1c2d4c912b1a4e19c4/x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/styles.tsx#L172-L175